static/webpack-config.js

```diff
@@ -26,6 +27,19 @@ module.exports = function () {
     });
   }
 
+  const cssrtlPlugins = [];
```

Comment:
nit: I believe this pattern with the array that gets pushed to was created because there can be multiple htmlPlugins. I don't think it's necessary for cssRTL, however it's pretty simple so I'm okay with keeping it as-is if you'd like. Another option would be to make the `hasRtlLocale` boolean available in this scope, and then after the plugins array is defined we can add the plugin if `hasRtrlLocale` is true

Reply:
I see what you mean. I think we should keep this pattern so it's easier to see everything that is included in `plugins` instead of adding on to it in another chunk of code later in the file. I renamed it to `cssRtlPlugin` to indicate there's only one if that's clearer.